app/views/mentee_application_mailer/notify_applicant_of_submission.html.erb

```diff
@@ -0,0 +1,15 @@
+<div style='display: block; max-width: 36rem; margin: 0 auto; padding-top: 4rem;'>
+  <p>Hey <%= @recipient.full_name %>,</p>
+
+  <p>We’ve received your application! Thanks so much for taking the time to apply for the Agency of Learning's <%= @application.cohort.name %> cohort.</p>
```

Comment:
note -- this mailer also leverages the cohort's name, so we should make the cohort names descriptive.